modules/ivc_champva/app/controllers/ivc_champva/v1/pega_controller.rb

```diff
@@ -39,6 +39,8 @@ def update_status
 
       private
 
+      # rubocop:disable Metrics/MethodLength
```

Comment:
Could you disable using `.rubocop_todo.yml` instead?

You can run `rubocop --auto-gen-config --exclude-limit 100`.

Reply:
@ericboehs  Done, but now rubocop does not seem to be picking up the exclusion.  What did I miss?

Reply:
This is borked! Looks like a bug in rubicop :(

Go ahead and revert the rubocop_todo.yml change. Feels free to nuke the commit:

```
git checkout CHAMPVA-95556-confirmation-email-bug
git reset --hard HEAD~1
git push --force-with-lease
```